physionet-django/events/templates/events/event_home.html

```diff
@@ -55,6 +55,26 @@ <h5 class="modal-title">Add Event</h5>
     </div>
   </div>
 
+    <div class="modal fade" id="edit-event-modal" tabindex="-1" role="dialog" aria-labelledby="edit-event-modal" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title">Edit Event</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="container">
+              <form action="" method="post" class="form-signin edit-event" >
+                {% csrf_token %}
+                {% include "form_snippet.html" with form=event_form %}
+                <button class="btn btn-primary btn-fixed" name="edit-event" type="submit">Edit event</button>
```

Comment:
I find the "Edit event" text on the button slightly odd, though can't explain why! Should we change it to something like "Save" instead?

Reply:
> This is great, thanks Amit!
> 
> Currently an error is raised when someone attempts to create/edit an event with an existing title (`IntegrityError at /events/. UNIQUE constraint failed: events_event.title, events_event.host_id`). Are you able to catch this and prevent the form being saved until the title is unique?
> 
> > Do we add a test for event create and update event now?(or should we wait till we add other features for events). probably now sound like a good idea, as it will be a simple test now and we can add more cases as we add features
> 
> Yeah, I think it would be good to add one or two tests now. Please could you add to the PR?

Thank you for the feedback. Working on it.